README.md

```diff
@@ -99,10 +99,13 @@ The config file (default file name: `.lintr`) is in [Debian Control Field Format
 - `exclude_start` - a regex pattern to start exclusion range. Default is "# nolint start"
 - `exclude_end` - a regex pattern to end exclusion range. Default is "# nolint end"
 
-An example file that uses 120 character line lengths, excludes a couple of
-files and sets different default exclude regexs follows.
+
+### .lintr File Example
+
+An example file that uses 120 character line lengths, excludes a couple of files, disables a specific lintr, and sets different default exclude regexs follows:
```

Comment:
this sentence is funny to me. I think it's better as

The following is an example .lintr file that...

the criteria would also be easier to parse as a bulleted list IMO

also the plural of regex is regexes IINM